external/test-runtime/optional.json

```diff
@@ -4,7 +4,9 @@
       "dependencies": {
         "Microsoft.DotNet.IBCMerge": "4.6.0-alpha-00001",
         "Microsoft.DotNet.UAP.TestTools": "1.0.2",
-        "TestILCNugetPackageForCoreFX": "1.0.0-beta-25304-01"
+        "TestILC.amd64ret": "1.0.0-beta-25305-00",
```

Comment:
Back when @mellinoe did his work to transition into `<PackageReference>` he removed all jsons except for the optional one. @mellinoe do you have an issue tracking removing that one as well?